parm/config/gfs/config.coupled_ic

```diff
@@ -21,7 +21,7 @@ case "${CASE}" in
     export CPL_WAVIC=GEFSwave20210528v2
     ;;
   "C768")
-    export CPL_ATMIC=HR1
+    export CPL_ATMIC=HR2
     export CPL_ICEIC=HR1
     export CPL_OCNIC=HR1
     export CPL_WAVIC=HR1
```

Comment:
So only the ATM ICs are being updated for HR2?

Reply:
Correct